### PR TITLE
Added Gen AI Quiz Application with Gemini LLM in Next JS

### PR DESCRIPTION
**Pull request by J-B-Mugundh**
Closes: #845

- **Title:** Gen AI Quiz Application with Gemini LLM in Next JS
- **Name:** J B Mugundh
- **Idenitfy yourself:** GSSOC-EXTD

### Describe the add-ons or changes you've made 📃

- Gen AI Quiz Application with Gemini LLM in Next JS

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Add 1 ss file and README file. Also enlist your project in Nextjs readme file

**Comment by J-B-Mugundh** [comment]
> Add 1 ss file and README file. Also enlist your project in Nextjs readme file

Ok sure! Lost track of this. Will finish it soon!

**Comment by J-B-Mugundh** [comment]
@Avdhesh-Varshney Please check now!

File: Next-JS-Projects/Advanced/Gen-AI-Quiz-Application/README.md
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/Advanced/Gen-AI-Quiz-Application/README.md:1]: image

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Enlist your project in Nextjs readme file

**Comment by J-B-Mugundh** [comment]
@Avdhesh-Varshney Updated!

File: Next-JS-Projects/README.md
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/README.md:1]: Create row 2 for your project and leave other 2 columns free.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Next-JS-Projects/README.md
Comment (J-B-Mugundh) [reply Next-JS-Projects/README.md:1]: > Create row 2 for your project and leave other 2 columns free.

Added!

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**